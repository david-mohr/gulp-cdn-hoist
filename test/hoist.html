<html>
<head>
  <!-- build:css /vendor.css -->
  <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
  <link href="//example.com/some/other.css" rel="stylesheet">
  <link href="/local.css" rel="stylesheet">
  <!-- endbuild -->
  <!-- build:css /app.css -->
  <link href="/local.css" rel="stylesheet">
  <!-- endbuild -->
  <!-- build:js /vendor.js -->
  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
  <script src="local.js"></script>
  <!-- endbuild -->
  <!-- build:js /app.js -->
  <script src="local.js"></script>
  <!-- endbuild -->
</head>
</html>
