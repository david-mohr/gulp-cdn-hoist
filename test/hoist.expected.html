<html>
<head>
  <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
<link href="//example.com/some/other.css" rel="stylesheet">
<!-- build:css /vendor.css -->
  
  
  <link href="/local.css" rel="stylesheet">
  <!-- endbuild -->
  
<!-- build:css /app.css -->
  <link href="/local.css" rel="stylesheet">
  <!-- endbuild -->
  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<!-- build:js /vendor.js -->
  
  <script src="local.js"></script>
  <!-- endbuild -->
  
<!-- build:js /app.js -->
  <script src="local.js"></script>
  <!-- endbuild -->
</head>
</html>
